openquake/qa_tests_data/ucerf/report.rst

```diff
@@ -162,13 +162,13 @@ Number of ruptures per tectonic region type
 source_model grp_id trt                  num_sources eff_ruptures weight
 ============ ====== ==================== =========== ============ ======
 ltbr0001     0      Active Shallow Crust 12          335          12    
-ltbr0002     1      Active Shallow Crust 12          277          12    
+ltbr0002     1      Active Shallow Crust 12          284          12    
 ============ ====== ==================== =========== ============ ======
 
 =============== ===
 #TRT models     2  
 #sources        24 
-#eff_ruptures   612
+#eff_ruptures   619
```

Comment:
This is actually the report for the event based, not for the classical calculator.